src/wp-includes/revision.php

```diff
@@ -516,6 +516,40 @@ function wp_get_post_revisions( $post_id = 0, $args = null ) {
 	return $revisions;
 }
 
+/**
+ * Returns the url for viewing and potentially restoring revisions of a given post.
+ *
+ * @since 5.9.0
+ *
+ * @param int|WP_Post $post_id Optional. Post ID or WP_Post object. Default is global `$post`.
+ * @return null|string The URL for editing revisions on the given post, otherwise null.
+ */
+function wp_get_post_revisions_url( $post_id = 0 ) {
+	$post = get_post( $post_id );
+
+	if ( ! $post instanceof WP_Post ) {
+		return null;
+	}
+
+	// If the post is a revision, return early.
+	if ( 'revision' === $post->post_type ) {
+		return get_edit_post_link( $post );
+	}
+
+	if ( ! wp_revisions_enabled( $post ) ) {
+		return null;
+	}
+
+	$revisions = wp_get_post_revisions( $post->ID, array( 'posts_per_page' => 1 ) );
+
+	if ( 0 === sizeof( $revisions ) ) {
```

Comment:
```suggestion
	if ( 0 === count( $revisions ) ) {
```

[`sizeof()`](https://www.php.net/manual/en/function.sizeof.php) is an alias of [`count()`](https://www.php.net/manual/en/function.count.php). Let's use the native instead of alias.